website/content/docs/agent/config/config-files.mdx

```diff
@@ -2071,6 +2071,12 @@ specially crafted certificate signed by the CA can be used to gain full access t
       * `TLSv1_2` (default)
       * `TLSv1_3`
 
+    - `verify_server_hostname` ((#tls_internal_rpc_verify_server_hostname)) When 
+      set to true, Consul verifies the TLS certificate presented by the servers 
+      match the hostname `server.<datacenter>.<domain>`. By default this is false, 
+      and Consul does not verify the hostname of the certificate, only that it 
+      is signed by a trusted CA. 
+
```

Comment:
I think we need to add `Overrides [`tls.defaults.verify_server_hostname`](#tls_defaults_verify_server_hostname).` at line 2187.